ansible/roles/private_clone/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Check if the directory already exists
+  stat:
+    path: /home/frappe/tmp/
```

Comment:
Set this to whatever you will realistically use in prod (eg /home/frappe/funder_management_system). Modify it to tmp in your local setup.